demos/serverless/README.md

```diff
@@ -61,6 +61,13 @@ npm run deploy -- -r us-east-1 -b <deploy-bucekt> -o <capture-bucket-prefix> -i
 
 Note that you need to enable these regions if you plan to use media capture. For more information, see [Managing AWS Regions](https://docs.aws.amazon.com/general/latest/gr/rande-manage.html).
 
+You can also use the live connector feature to broadcast the captured artifacts. The default layout is vertical layout. You can specify the ivs channel prefix with --ivs-channel-prefix option, for example
```

Comment:
Is there a public documentation on this feature? Suggest adding relevant mark down links here to help builders better understand the feature such as a blog post, AWS docs etc.

If `ivs channel` is Amazon IVS suggest elaborating a bit here. Instead of just ivs channel, "Amazon Interactive Video Service (IVS) channel prefix....".

Also, its unclear whether the channel will be internally created or how. From request handling looks like we dont create it. Basically elaborating more here will help builders as well.

Reply:
I will also add description about how to use the playback url to view the broadcast